tests/framework/helpers/ArrayHelperTest.php

```diff
@@ -51,6 +51,9 @@ protected function setUp()
 
     public function testToArray()
     {
+        $dataArrayable = $this->getMock('yii\\helpers\\ArrayHelper');
+        $this->assertEquals([], ArrayHelper::toArray($dataArrayable));
```

Comment:
what are you testing here? converting empty object to an empty array? if so, `new stdClass` would be more intuitive than calling getMock() for it.